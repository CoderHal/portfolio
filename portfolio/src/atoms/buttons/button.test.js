import React from "react";
import Renderer from "react-test-renderer";
import { ThemeProvider } from "styled-components";
import Button from "./index";

const theme = {
  mode: "light"
};

it("shows difference between 2 buttons", () => {
  const btn = Renderer.create(
    <ThemeProvider theme={theme}>
      <Button color="primary" />
    </ThemeProvider>
  ).toJSON();
  expect(btn).toMatchSnapshot();
});
