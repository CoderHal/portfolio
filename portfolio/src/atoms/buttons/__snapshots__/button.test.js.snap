// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`shows difference between 2 buttons 1`] = `
<div
  className="sc-AxjAm elrVCD"
  color="primary"
/>
`;
